aalto/zulip.rst

```diff
@@ -0,0 +1,75 @@
+========
+Zulip
+========
+
+Introduction
+------------
+
+CS-IT can host `Zulip <https://zulipchat.com/>`_ chat instances e.g. for your courses. These chat instances are hosted at ``<chat-name>.zulip.cs.aalto.fi``. Login to the chats uses Aalto credentials (with possible support for extenal users later on).
+
+In order to request a chat instance, please send an email to ``guru@cs.aalto.fi``.
+
+.. note::
+
+   There are also `Rocket.Chat <https://rocket.chat/>`_ instances available, provided by ITS. Rocket.Chat instances can be requested from ``servicedesk@aalto.fi``
+
+Configuring your organization
+------------------------------------
+
+Below are listed the most important settings found under *Manage organization* in Zulip. There is no easy way for us to enforce these, so it is your responsibility as organization owner or admin to make sure they are set correctly. Make sure any owners/admins you appoint are aware of these as well.
+
+.. note::
+
+    Settings that are not mentioned here, you can configure to your liking. However you should still exercise care.
+
+
+**Organization settings / Video chat provider**
+
+  * Set to ``None``
+  * The default provider (Jitsi) has not been evaluated or approved by Aalto
+  * Integration with Aalto Zoom may come later on
+
+
+**Organization permisisons / Invitation settings**
+
+We cannot allow people to register freely using random emails, therefore:
```

Comment:
Is this because of Aalto policies saying that we can't/aren't allowed to provide services to random people?  If so, we could say that so it's clear why.

Reply:
Not an Aalto policy, but common sense. Registering with just an email address is an open invitation for bots etc.